prompts.md

```
@@ -1,122 +1,67 @@
-7. [Pull requests](#7-pull-requests)
-
----
-
-## 1. Descripción general del producto
-
-**Prompt 1:**
-
-**Prompt 2:**
-
-**Prompt 3:**
+1. [Descripción general del producto y Arquitectura del Sistema](#1-Descripción-general-del-producto-y-Arquitectura-del-Sistema)
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)